### PR TITLE
educationVo updated

### PR DESCRIPTION
**Pull request by jitendrajaria**
educationVo was not working as expected when multiple education fields are provided, fixed in this PR

### CODE REVIEW

File: src/main/java/creativei/entity/InquiryEducation.java [hunk 1]
```diff
@@ -26,24 +29,20 @@
...
-            this.status = EducationStatus.stringToEnum(educationVo.get(i).getStatus());
-            this.year = educationVo.get(i).getYear() == null ? null : educationVo.get(i).getYear();
-            this.aggregateMarks = educationVo.get(i).getAggregateMarks() == null ? null : educationVo.get(i).getAggregateMarks();
-            this.markScheme = MarkScheme.stringToEnum(educationVo.get(i).getMarkScheme());
-            this.type = educationVo.get(i).getType() == null || educationVo.get(i).getType().isEmpty() ? null : educationVo.get(i).getType();
-            iterator.next();
-            i++;
-        }
+    public InquiryEducation(EducationVo educationVo) {
+
```
Comment (aayushKhandpur) [inline src/main/java/creativei/entity/InquiryEducation.java]: Remove blank line

File: src/main/java/creativei/helper/ResponseHelper.java [hunk 2]
```diff
@@ -70,8 +70,8 @@ private static AddressVo getCreateAddressResponseData(InquiryAddress inquiryAddr
             educationVo.setYear(inquiryEducation.get(i).getYear() == null ? null : inquiryEducation.get(i).getYear());
             educationVo.setType(inquiryEducation.get(i).getType() == null ? null : inquiryEducation.get(i).getType());
             iterator.next();
-            i++;
             educationVos.add(educationVo);
+            i++;
```
Comment (aayushKhandpur) [inline src/main/java/creativei/helper/ResponseHelper.java]: Can we use for each loops instead?

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address comments

File: src/main/java/creativei/entity/InquiryEducation.java (hunk 1 above)
Comment (jitendrajaria) [reply src/main/java/creativei/entity/InquiryEducation.java]: Issue resolved in PR 8da9452

File: src/main/java/creativei/helper/ResponseHelper.java (hunk 2 above)
Comment (jitendrajaria) [reply src/main/java/creativei/helper/ResponseHelper.java]: yes, I have tried it in commit 8da9452 please review

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**